Upper Level BCIO/bcio_upper_level.owl

```diff
@@ -746,7 +747,7 @@
     <!-- http://humanbehaviourchange.org/ontology/BCIO_045000 -->
 
     <owl:Class rdf:about="http://humanbehaviourchange.org/ontology/BCIO_045000">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000011"/>
```

Comment:
A few lines up, the class 'style of delivery', which actually hasn't changed in this release in this file, now needs to change status. It is still listed as 'proposed' (along with one or two other related classes). However, the style of delivery spreadsheet has recently been updated and all classes in it have now got curation status 'published'. That means that these classes, also represented here, are getting double curation statuses in the merged file.  This is an other example of something that in the long term we would solve pre-release. However, for now could you update the status in this one to 'published' ?

Reply:
I don't think I understand. The "BCI style of delivery" is marked as "Published" and in the final owl file I also see only one curation status annotation.

Reply:
It was 'communication style' not 'BCI style of delivery'. I think it is the parent? 

<img width="583" alt="Screenshot 2023-11-21 at 11 31 44" src="https://github.com/HumanBehaviourChangeProject/ontologies/assets/512096/fd589991-c059-495c-9ff6-50e8d8cf0529">

These  three classes were potentially affected by this.  
<img width="977" alt="Screenshot 2023-11-21 at 11 32 36" src="https://github.com/HumanBehaviourChangeProject/ontologies/assets/512096/845cb1d2-e2f1-4210-ba3d-42e34d29cc2c">

Reply:
Should be fixed now.

FYI: Just changing it to proposed and rerun the release script still produced both "Proposed" and "Published". The issue was that the subontologies import `http://humanbehaviourchange.org/ontology/bcio_upper_level.owl` and in the final merge command robot resolves imports relative to each input file. But as the new `bcio_upper_level.owl` is not in the same directory as the subontologies but rather in the `scripts` directory, robot downloads the upper level ontology from the web and hence uses an older version which still contained the `'has curation status' Proposed`.